README.md

```diff
@@ -143,6 +143,7 @@ Please ⭐️ this repo if you find it useful
 | indicators: `name:source:attribute` | string | optional | v1.0.1 | entity attribute
 | indicators: `name:source:mapper` | function | optional | v1.0.1 | value processing function
 | indicators: `name:tap_action` | [action object](#tap-action-object) | true | v1.1.0 | Action on click/tap.
+| indicators: `name:disabled` | function | optional | v2.4.5 | disabled/enabled rendering of indicators
```

Comment:
let's use `hide` instead and change the description accordingly

Reply:
I don't see it used anywhere in the indicators context, and if it already used, lets expend it to be either boolean or function